filebeat/docs/running-on-kubernetes.asciidoc

```diff
@@ -112,6 +112,36 @@ oc patch namespace kube-system -p \
 This command sets the node selector for the project to an empty string. If you
 don't run this command, the default node selector will skip master nodes.
 
+[float]
```

Comment:
See my comments about CloudFoundry and running Metricbeat on k8s. They apply here, too.